File: app/src/main/java/com/softdev/smartraysam/carfeeds/model/Cars.java
package com.softdev.smartraysam.carfeeds.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Ashish on 30-01-2017.
 */

public class Cars {
    @SerializedName("placemarks")
    private List<carModel> car = null;

    public List<carModel> getCarModel() {
            return car;
        }
        public void setCar(List<carModel> car) {
            this.car = car;
        }
    }

File: app/src/main/java/com/softdev/smartraysam/carfeeds/model/Cars.kt
package com.softdev.smartraysam.carfeeds.model

import com.google.gson.annotations.SerializedName


/**
 * Created by Smart Raysam on 15-09-2018.
 */

class Cars {
    @SerializedName("placemarks")
    var carModel: List<carModel>? = null
        private set

    fun setCar(car: List<carModel>) {
        this.carModel = car
    }
}

